src/sass/accordion/accordion.scss

```diff
@@ -0,0 +1,63 @@
+@import "../variables/variables";
+@import "../mixins/private/token-mixins";
+
+ul.accordion {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+}
+
+ul.accordion li:not(:last-child) {
+    border-bottom: 1px solid var(--color-stroke-subtle);
+}
+
+ul.accordion summary.details__summary {
+    border-radius: 0;
+    display: flex;
+    font-size: var(--font-size-medium);
+    justify-content: space-between;
+    min-height: 48px;
+    padding: 12px 16px;
+}
+
+ul.accordion details .details__content {
+    margin: 0 16px 6px;
+    opacity: 0;
+    transform: scaleY(0);
+    transform-origin: top;
+    transition:
+        opacity 500ms cubic-bezier(0.3, 0, 0, 1),
+        transform 500ms cubic-bezier(0.3, 0, 0, 1);
+}
+
+ul.accordion details[open] .details__content {
+    opacity: 1;
+    transform: scaleY(1);
+    transition:
```

Comment:
As per spec, there are two animations @ArtBlue, one on open and another on close

Reply:
So, there are many issues with that.

First and most important, the collapse animation can't work the way it's set up. Actually, there is NO possible way to do collapse animation with our current approach because the standard html `details` collapse is immediate. This solution was a pure `css` way to simulate expansion animation, but the same cannot be used to collapse. The reason it works with expand is the expand of the `details` is immediate and then the transition takes time to complete when the details is expanded.

The only way to do animation with collapse is to intentionally delay the collapse to allow the animation to complete before allowing a collapse - this would be extremely hacky and IMO very unnecessary, and would look pretty bad.

Let's remove the collapse transition because it's not doing anything ATM.

Reply:
Thats a good point @ArtBlue.